src/components/forms/CreateFurnitureForm.vue

```diff
@@ -56,7 +56,7 @@ export default {
           isUpdating: false,
           amountOfSeparators: 0,
           typeOfSeparators: 'centered',
-          spaces: [{ id: 1, width: 100 }],
+          spaces: [{ id: 0, width: 100 }],
```

Comment:
it's kind of weird to have a hardcoded id here but as you don't have any backend, let's keep it like that for now (except if you see an easy way to improve it)